release-pr/src/neon_release_pr/git.py

```diff
@@ -161,10 +162,54 @@ def rc_branch_name() -> str:
 
 
 def release_branch_name() -> str:
-    if ctx.component == "storage":
-        return "release"
-    else:
-        return f"release-{ctx.component}"
+    return release_branch_name_override() or f"release-{ctx.component}"
+
+
+def release_branch_name_override() -> Optional[str]:
+    overrides = {
+        "neondatabase/neon": {
+            "storage": "release",
+        },
+        "databricks-eng/hadron": {
+            "storage": "neon-release-storage",
+            "compute": "neon-release-compute",
+            "proxy": "neon-release-proxy",
+            "hcm": "release-compute",
+            "hadron": "release",
+        },
+    }
+
+    repo = github_repo()
+
+    if repo is None:
+        return None
+
+    repo_override = overrides.get(repo, None)
+
+    if repo_override is None:
+        typer.echo(f"[info] no repo override found for repo {repo}")
+        return None
+
+    return repo_override.get(ctx.component, None)
```

Comment:
nit:
```suggestion
    if repo not in overrides:
        typer.echo(f"[info] no repo override found for repo {repo}")
        return None
        
    return overrides[repo].get(ctx.component, None)
```

Reply:
This is good, applied the first suggestion to keep the debug line.